GREEDY/LogHandler.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace GREEDY
+{
+    public class LogHandler : DelegatingHandler
+    {
+        private static object lockObject = new object();
+        private string logEntrySeperator => "---------------------------------------------------";
+
+        protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+        {
+            HttpResponseMessage response=null;
+            try
+            {
+                StringBuilder builder = new StringBuilder();
+                builder.AppendLine(logEntrySeperator);
+                builder.AppendLine("Request");
+                builder.AppendLine(DateTime.Now.ToString("yyyy/mm/dd hh:mm:ss"));
```

Comment:
change it to " ToString("g", CultureInfo.CreateSpecificCulture("en-us")) " or just g without specific culture